api/client/client.go

```diff
@@ -611,6 +612,31 @@ func (c *Client) LoginRuleClient() loginrulepb.LoginRuleServiceClient {
 	return loginrulepb.NewLoginRuleServiceClient(c.conn)
 }
 
+// TrustClient returns an unadorned Trust client, using the underlying
+// Auth gRPC connection.
+func (c *Client) TrustClient() trustpb.TrustServiceClient {
+	if c.isClosed() {
+		return trustpb.NewTrustServiceClient(closedClientConnection{})
+	}
+
+	return trustpb.NewTrustServiceClient(c.conn)
+}
+
+// closedClientConnection is a [grpc.ClientConnInterface] implementation
+// that returns a connection error when a unary RPC is invoked or a new
+// stream is attempted to be created.
+type closedClientConnection struct{}
+
+// Invoke implements [grpc.ClientConnInterface]
+func (g closedClientConnection) Invoke(ctx context.Context, method string, args interface{}, reply interface{}, opts ...grpc.CallOption) error {
+	return trace.ConnectionProblem(nil, "connection was closed when attempting to call %s", method)
+}
+
+// NewStream implements [grpc.ClientConnInterface]
+func (g closedClientConnection) NewStream(ctx context.Context, desc *grpc.StreamDesc, method string, opts ...grpc.CallOption) (grpc.ClientStream, error) {
+	return nil, trace.ConnectionProblem(nil, "connection was closed when attempting to call %s", method)
+}
```

Comment:
nit: IIUC the purpose of this type is to be a placeholder for building sub-clients from an already closed root client.  I'd clarify that purpose on the comments, and update the error message to indicate that the client itself was explicitly closed.

"connection was closed while [...]" feels like it could be a network error, or possibly something the remote server did, but something like "cannot call <method> after client has already been closed" would make me immediately check where I call `Client.Close`.  Easier to debug.